src/lib/Strategy/FocusMode/ContentStructureRedirectStrategy.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\Strategy\FocusMode;
+
+use Ibexa\Contracts\AdminUi\FocusMode\RedirectStrategyInterface;
+use Ibexa\Contracts\Core\Repository\LocationService;
+use Ibexa\Contracts\Core\SiteAccess\ConfigResolverInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+final class ContentStructureRedirectStrategy implements RedirectStrategyInterface
```

Comment:
The reason for this class is highly specific - it is responsible for changing the redirection target **for URI's that are not available in focus mode in particular**. This information is nowhere to be found in the class, and I think it should be at least provided as a comment (if not reflected in the class name).

Because I think that ultimately what this is about is not what it does, but why it does it. And someone looking at it in a few months might not easily realize what it's for.

Reply:
Went for a comment via https://github.com/ibexa/admin-ui/pull/1065/commits/989b539431d4fa2879c7530fcb4e8a2765c81299.